hpc/app/workload_managers.yml

```diff
@@ -43,8 +43,42 @@ node_types:
         required: false
         type: string
         default: ""
+      ssh_host: 
+        required: true
+        type: string
+      hpc_label:
+        required: false
+        type: string
+        default: ""
+      scrape_interval:
+        required: false
+        type: integer
+        default: 15
     capabilities:
       resources:
         type: sodalite.capabilities.WM.JobResources
       optimisations:
-        type: sodalite.capabilities.OptimisedTarget
\ No newline at end of file
+        type: sodalite.capabilities.OptimisedTarget
+    interfaces:
+      Standard:
+        type: tosca.interfaces.node.lifecycle.Standard
+        operations:  
+          start:
+            inputs:
+              scheduler:            { type: string, default: { get_property: [SELF, scheduler] } }
+              ssh_host:             { type: string, default: { get_property: [SELF, ssh_host] } }
+              hpc_label:            { type: string, default: { get_property: [SELF, hpc_label] } }
+              monitoring_id:        { get_input: monitoring_id }
+              deployment_label:     { get_input: deployment_label}
+              jwt:                  { get_input: jwt }
+              hpc_exporter_address: { get_input: hpc_exporter_address }
+            implementation: 
+              primary: playbooks/workflows/create_collector.yml
```

Comment:
This path is wrong, as you placed the collector playbooks in the playbooks/ directory...